### ISSUE TITLE
Less obtrusive notifications.

### ISSUE THREAD

**Issue by simar7** [sveale/remote-edit#119]
Hello,

I've using this package and it works great. However I feel the notifications could be slightly tweaked to provide a much better user experience. Currently while editing files, a total of 4 notifications are shown while editing and saving a remote file. IMHO, this could be either made configurable to only show when the file is successfully saved or when any errors/exceptions occur.

Let me know if you have something better in mind and also if you need a hand in implementing this.

Thanks!

**Comment by sveale** [comment]
Hi! Good to hear :)

There is a settings option for turning notifications completely off.

Additionally, I think there is work being done on [atom-notifications](https://github.com/atom/notifications) to suppress or group together notifications.

I think it's a good idea to use verbosity levels to allow the user to choose the types of notifications he should get. I'm not entirely sure what these levels should be. It's probably easier to just implement a setting that only prints errors/exceptions and when a file have been successfully uploaded.

I'll add the enhancement label to the issue. I'll probably not get around to implementing it for quite some time. Any help you could lend would be greatly appreciated.

**Comment by BertLindeman** [comment]
I agree with the popping-up of messages for a simple remote update if all goes OK.
Less is better in that case.

Maybe related (if not, please let me know) 
I have been using atom.io just from today, so no solid user in this area yet. 
In the phase where you setup a new host connection, I miss some kind of log to SEE what happens during the connect. All I now see is part of a message overlayed with the search field.
Created a wrong host on purpose, trying to show the situation:
![remoteedit](https://cloud.githubusercontent.com/assets/8142081/12273894/9334fd2c-b968-11e5-876f-c875a4c36b5e.png)

The error message is not the first message. The first one is hidden completely.
